Library/Homebrew/formula.rb

```diff
@@ -2423,10 +2423,30 @@ def depends_on(dep)
       specs.each { |spec| spec.depends_on(dep) }
     end
 
+    # This is used for dependencies that are provided by macOS.
+    # No-op on macOS, as we use the system libraries there.
+    # On Linux this will add the brewed version of the library to the
+    # dependency tree.
```

Comment:
```suggestion
    # Indicates use of dependencies provided by macOS.
    # On macOS this is a no-op (as we use the system libraries there).
    # On Linux this will act as `depends_on`.
```